src/bluetooth.hpp

```diff
@@ -0,0 +1,63 @@
+#ifndef BLUETOOTH
+#define BLUETOOTH
+
+#include "wrap-hwlib.hpp"
+
+class bluetooth {
+public:
+     /// \brief
```

Comment:
Incorrect use of Doxygen, please read the wiki for the correct Doxygen standard.